src/main/java/de/tum/in/www1/artemis/repository/ResultRepository.java

```diff
@@ -41,6 +41,9 @@
     @EntityGraph(type = LOAD, attributePaths = "submission")
     List<Result> findByParticipationExerciseIdOrderByCompletionDateAsc(Long exerciseId);
 
+    @EntityGraph(type = LOAD, attributePaths = { "submission", "feedbacks" })
+    List<Result> findWithEagerSubmissionAndFeedbackByParticipationExerciseId(Long exerciseId);
```

Comment:
```suggestion
    List<Result> findWithEagerSubmissionAndFeedbackByExerciseId(Long exerciseId);
```

Reply:
This should need the Participation to magically join and have an exercise id for comparison.

Reply:
@jpbernius  ah, okay. I did not know that.  I'll add the Participation too..

Reply:
No, I mean your name should be fine as is ;)

Reply:
ah, alright 😄